src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -2938,7 +2960,9 @@ namespace wallet_rpc
       BEGIN_KV_SERIALIZE_MAP()
         KV_SERIALIZE    (type);
         KV_SERIALIZE    (name);
-        KV_SERIALIZE    (value);
+        KV_SERIALIZE_OPT(value, std::string(""));
+        KV_SERIALIZE_OPT(owner, std::string(""));
+        KV_SERIALIZE_OPT(backup_owner, std::string(""));
         KV_SERIALIZE_OPT(signature, std::string(""));
```

Comment:
I'm fairly sure (but not 100%) that none of these need to be `_OPT`: I think it is only there for when you want a non-default-constructed value if the field is omitted.

Reply:
Yeah, that's correct. I have removed the _OPT part.